.github/workflows/build-test

```diff
@@ -51,6 +51,8 @@ cd ${GITHUB_WORKSPACE}/tests
 
 python -m pip install --pre -r test-requirements.txt
 
+python -m pip install --upgrade --pre pytket
```

Comment:
We should probably restrict to pytket 1.x.

Reply:
Good point